src/autogluon/bench/frameworks/tabular/setup.sh

```diff
@@ -7,11 +7,11 @@ if [ ! -d $DIR ]; then
 fi
 
 # create virtual env
-python3.8 -m venv $DIR/.venv
+python3.9 -m venv $DIR/.venv
```

Comment:
Is there a reason why we had `3.8` here? If we're going to pick one, we might as well choose `3.9`.